<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ship</class>
 <widget class="QMainWindow" name="Ship">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ship</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>974</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit Game</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="eastButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GO EAST</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tempText">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="northButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GO NORTH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="westButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GO WEST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="southButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GO SOUTH</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="mapOfShip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>731</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="itemList">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="backpackButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>380</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Backpack</string>
    </property>
   </widget>
   <widget class="QLabel" name="supplyRoomLbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Supply Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="weaponsRoomLbl">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Weapons Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="hallwayLbl">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hallway</string>
    </property>
   </widget>
   <widget class="QLabel" name="sleepingQuartersALbl">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sleeping Quarters A</string>
    </property>
   </widget>
   <widget class="QLabel" name="engineRoomLbl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Engine Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="upstairsHallwayLbl">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upstairs Hallway</string>
    </property>
   </widget>
   <widget class="QLabel" name="coalRoomLbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coal Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="kitchenLbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kitchen</string>
    </property>
   </widget>
   <widget class="QLabel" name="sundeckLbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sundeck</string>
    </property>
   </widget>
   <widget class="QLabel" name="captainsQuartersLbl">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Captains Quaeters</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
